src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -70,14 +70,15 @@ export interface StoragePoolFormValues {
   powerflex_sdt?: string;
   powerflex_user_name?: string;
   powerflex_user_password?: string;
+  poolSizePerClusterMember?: ClusterSpecificValues;
   pure_api_token?: string;
   pure_gateway?: string;
   pure_gateway_verify?: string;
   pure_mode?: string;
   readOnly: boolean;
   size?: string;
   source: string;
-  sourcePerClusterMember?: ClusterSpecificValues;
+  poolSourcePerClusterMember?: ClusterSpecificValues;
```

Comment:
I think the pool prefix is redundant. This is the `StoragePoolFormValues` type after all.